src/content/reference/cli.mdx

```diff
@@ -604,6 +604,22 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
 | :-------- | :----: | :------------------------------------------------------------------------------------- |
 | _--label_ | (list) | tag and organize preview environments using labels (multiple _--label_ flags accepted) |
 
+#### sleep
+
+Sleeps an okteto preview environment. You need to provide that preview name as an argument to the command.
+
+```console
+$ okteto preview sleep [name]
+```
+
```

Comment:
@andreafalzetti that is the only resource we have on this today. Agree it needs to be improved. For now, let's link to that one. I'll add an issue on improving that page.

Reply:
I am not sure linking this to the previews is completely ok, this article talks about resources at a namespace, and this feat is for previews, which are namespaces, but we should then explain this too? 

@pchico83 WDYT

Reply:
My vote on something like this is: let's merge this unless anyone objects and thinks this is a blocking suggestion.

Since I'm actively shifting my focus to our documentation we should be able to make quality of life improvements like this in follow-on PRs 😄

Reply:
Ship it!

Reply:
Added link for article